### DISCUSSION TITLE
v0.9.1

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#1792] [General]
Bugfix release for v0.9.0

- Fixed crash when closing the parent patch of an opened subpatch
- Fixed plugin shutdown crash on Windows
- Fixed array redrawing bug
- Fixed crash when using palettes or "add object" menu in plugins
- Fixed broken multi-drag connection behaviour
- Fixed graph rendering bug when scrolling
- Fixed rendering artifacts on connection cords
- Fixed [pic] send symbol
- Fixed multichannel connection rendering
- Fixed MIDI input for Logic instrument plugins
- Fixed strange tooltip behaviour on Linux
- FIxed crash when sending "open" message to "pd"
- Fix redo broken after deleting an object and undoing
- Fix rendering with very small global scale applied
- Fixed memory leak when plugin editor is closed
- Fix numbox issue with canvas synchronisation
- Fixed window dragging scale bug
- Optimised [note] and [messbox] objects
- Drastically reduced VRAM usage
- Optimised various actions like copy/paste/duplicate/remove when performed on many objects
- Small startup optimisations
- Various rendering optimisations 
   - More efficient use of memory and CPU cache
   - Do more work directly from shader
   - Removed redundant work
- Fixed various iOS bugs (release will be on AppStore soon!)

<hr /><em>This discussion was created from the release <a href='https://github.com/plugdata-team/plugdata/releases/tag/v0.9.1'>v0.9.1</a>.</em>

**Comment by zayaman38** [comment]
Hi, I've been having trouble with this version using the Gem library. Any suggestions? I'm specifically using Wingem, and pd crashes. Operating system (Windows 10). I would greatly appreciate it.

**Reply by dromer** [reply]
@zayaman38 you can report tickets on the issues page, but first check if there is already a ticket you can add information to: https://github.com/plugdata-team/plugdata/issues?q=is%3Aissue%20state%3Aopen%20gem

**Reply by timothyschoen** [reply]
Hi! Gem is not supported for v0.9.1, but if you download the latest nightly build of v0.9.2, it comes with Gem built in!
